src/libraries/Microsoft.PowerFx.Interpreter/Functions/StandardErrorHandling.cs

```diff
@@ -369,24 +432,29 @@ private static List<List<T>> Transpose<T>(List<List<T>> columns, int columnSize)
             {
                 var resultRows = new List<DValue<RecordValue>>();
 
-                (var maxSize, var emptyTablePresent) = AnalyzeTableArguments(args);
+                (var maxSize, var minSize, var emptyTablePresent) = AnalyzeTableArguments(args);
                 if (maxSize == 0 || (emptyTablePresent && !transposeEmptyTable))
                 {
                     // maxSize == 0 means there are no tables with rows. This can happen when we expect a Table at compile time but we recieve Blank() at runtime,
                     // or all tables in args are empty. Additionally, among non-empty tables (in which case maxSize > 0) there may be an empty table,
                     // which also means empty result if transposeEmptyTable == false.
                     // Return an empty table (with the correct type).
+                    if (!emptyTablePresent)
+                    {
+                        return FormulaValue.NewBlank();
```

Comment:
>return FormulaValue.NewBlank(); [](http://example.com/codeflow?start=24&length=31)

This code will never be hit. If !emptyTablePresent, that means that maxSize == 0 (otherwise we wouldn't get in this if statement). But in AnalyzeTableArguments, emptyTablePresent will be set to true if there is at least one table with size = 0. And since maxSize = 0, this will always be true.

Reply:
And if this is indeed the case, we don't need the `transposeEmptyTable` argument anymore

Reply:
And if we don't need that, we probably don't need the `emptyTablePresent` from AnalyzeTableArguments either

Reply:
This code is hit when we have Blank as one of the arg.
e.g. Concatenate(If(1<0,["hello", "hi"]), " world") => Concatenate(Blank(), "World") => Blank()
Here, maxSize = 0 and emptyTablePresent = False (since we don't consider Blank() as empty table)

Reply:
I think I see. But if any of the table arguments is blank, we want to return blank as well. For example, those expressions should all return blank values:
- Concatenate(Filter(["a","b"], Len(Value)>10), If(1<0,["abcdef", "ghijkl"])) // an empty table + a blank table
- Find(If(1<0,["a","b"]), Filter(["abcdef", "ghijkl"], , Len(Value)>10)) // a blank table + an empty table

Reply:
Updated code to accommodate this behavior.